vita3k/gui/src/app_context_menu.cpp

```diff
@@ -97,6 +98,57 @@ static const char OS_PREFIX[] = "open ";
 static const char OS_PREFIX[] = "xdg-open ";
 #endif
 
+static std::map<std::string, std::map<std::string, time_t>> last_time_apps_used;
+
+void get_last_time_apps_used(HostState &host) {
```

Comment:
If the plan is to move the upper variable to GuiState, you can pass another argument

```suggestion
void get_last_time_apps_used(GuiState &gui, HostState &host) {
```

Reply:
Yes i know’ but for moment i keep like that